doc/debug-logs.md

```diff
@@ -0,0 +1,57 @@
+# Rules for debug logging
+
+- Do not log errors that bubble up to the controller runtime. They will be logged anyways and we do not want to log errors multiple times because it's confusing.

```

Comment:
```suggestion
- Do not log errors that bubble up to the controller runtime. They will be logged anyway, and we don't want to log errors multiple times because it's confusing.

```